gc/base/standard/ConfigurationStandard.cpp

```diff
@@ -87,6 +87,25 @@ MM_ConfigurationStandard::initialize(MM_EnvironmentBase* env)
 	return result;
 }
 
+
+void
+MM_ConfigurationStandard::initializeGCThreadCount(MM_EnvironmentBase* env)
+{
+	MM_GCExtensionsBase* extensions = env->getExtensions();
```

Comment:
This line needs to mo e down into the `#if defined`

Reply:
fixed